### PR TITLE
Updating pom to use new 1.1.4 parent-pom

### PR DESCRIPTION
**Pull request by zachschultz**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -5,11 +5,11 @@
   <parent>
     <groupId>io.dropwizard-bundles</groupId>
     <artifactId>parent-pom</artifactId>
-    <version>1.0.5</version>
+    <version>1.1.4-SNAPSHOT</version>
```
Comment (alexwen) [inline pom.xml]: The parent here should be the release version of Dropwizard, `1.1.4`.

File: pom.xml [hunk 2]
```diff
@@ -5,11 +5,11 @@
   <parent>
     <groupId>io.dropwizard-bundles</groupId>
     <artifactId>parent-pom</artifactId>
-    <version>1.0.5</version>
+    <version>1.1.4</version>
   </parent>
 
   <artifactId>dropwizard-version-bundle</artifactId>
-  <version>1.0.6-SNAPSHOT</version>  <!-- Make sure to keep this in sync with the parent pom version above. -->
+  <version>1.1.4</version>  <!-- Make sure to keep this in sync with the parent pom version above. -->
```
Comment (alexwen) [inline pom.xml]: The non-release version just be `1.1.4-SNAPSHOT`.

**Review by alexwen — CHANGES_REQUESTED** [review]

**Comment by alexwen** [comment]
@zachschultz looks like this build is failing due to using a deprecated method:
https://travis-ci.org/dropwizard-bundles/dropwizard-version-bundle/builds/285665950?utm_source=github_status&utm_medium=notification

We will want to stop using the Throwables method flagged by the build.

**Comment by bbeck** [comment]
Closing in favor of #15.

**Pull request closed without merging**